nvflare/private/fed/server/server_app_runner.py

```diff
@@ -66,13 +64,13 @@ def start_server_app(self, workspace: Workspace, server, args, app_root, job_id,
             logger.exception(f"FL server execution exception: {secure_format_exception(e)}")
             raise e
         finally:
-            self.update_job_run_status(server)
+            # self.update_job_run_status(server)
             server.status = ServerStatus.STOPPED
             server.engine.engine_info.status = MachineStatus.STOPPED
             server.stop_training()
 
     def sync_up_parents_process(self, args, server):
-        server.engine.create_parent_connection(int(args.conn))
+        # server.engine.create_parent_connection(int(args.conn))
```

Comment:
we should remove the commented out codes

Reply:
Removed.